### PR TITLE
Feature/main screen

### PR DESCRIPTION
**Pull request by AngelaLee18**
STORY
Used coordinator and created Main Screen

DESCRIPTION
Used coordinator to move from splash screen to main screen and designed Now Playing, Top Rated and Upcoming tab with a header "Movies DB" and search icon. 

EVIDENCE

https://user-images.githubusercontent.com/109097550/196446357-73357236-8139-4eb1-ac81-eb0e0c99a30c.mov

Screen Shot 2022-10-18 at 10 41 06

### CODE REVIEW

**Review by agustingodoyc — APPROVED** [review]

**Comment by RafaBenaGlobant** [comment]
There is some duplicated code between NowPlaying, TopRated and Upcoming VCs Have you considered an alternative?

File: TheMovieDB/TheMovieDB/Presentation/View/Main Screen/TopRatedViewController.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        checkNetworking()
+    }
+    
+    func checkNetworking() {
+        guard networkStatus == false else {
+            networkErrorImage.isHidden = true
+            return
+        }
+        networkErrorImage.isHidden = false
+    }
```
Comment (ezequielvelezglobant) [inline TheMovieDB/TheMovieDB/Presentation/View/Main Screen/TopRatedViewController.swift]: duplicated code

File: TheMovieDB/TheMovieDB/Presentation/View/Main Screen/UpcomingViewController.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        checkNetworking()
+    }
+    
+    func checkNetworking() {
+        guard networkStatus == false else {
+            networkErrorImage.isHidden = true
+            return
+        }
+        networkErrorImage.isHidden = false
+    }
```
Comment (ezequielvelezglobant) [inline TheMovieDB/TheMovieDB/Presentation/View/Main Screen/UpcomingViewController.swift]: duplicated code

**Review by ezequielvelezglobant — CHANGES_REQUESTED** [review]
as @RafaBenaGlobant mentioned there is duplicated code.
Think of a solution and tell us on slack!

File: TheMovieDB/TheMovieDB/Presentation/View/Main Screen/TopRatedViewController.swift (hunk 1 above)
Comment (AngelaLee18) [reply TheMovieDB/TheMovieDB/Presentation/View/Main Screen/TopRatedViewController.swift]: d544b15cef07f5f6da9e8d2b762dd3fb3ef0803c

File: TheMovieDB/TheMovieDB/Presentation/View/Main Screen/UpcomingViewController.swift (hunk 2 above)
Comment (AngelaLee18) [reply TheMovieDB/TheMovieDB/Presentation/View/Main Screen/UpcomingViewController.swift]: d544b15cef07f5f6da9e8d2b762dd3fb3ef0803c

File: TheMovieDB/TheMovieDB/Presentation/View/Main Screen/TopRatedViewController.swift (hunk 1 above)
Comment (AngelaLee18) [reply TheMovieDB/TheMovieDB/Presentation/View/Main Screen/TopRatedViewController.swift]: 187c21c7805f731d3ed2389271edf484eee5ae1b

File: TheMovieDB/TheMovieDB/Presentation/View/Main Screen/UpcomingViewController.swift (hunk 2 above)
Comment (AngelaLee18) [reply TheMovieDB/TheMovieDB/Presentation/View/Main Screen/UpcomingViewController.swift]: 187c21c7805f731d3ed2389271edf484eee5ae1b

**Review by RafaBenaGlobant — APPROVED** [review]

**Review by agustingodoyc — APPROVED** [review]

**Review by ezequielvelezglobant — APPROVED** [review]

**Pull request merged**